src/router/PrivateRouter.tsx

```diff
@@ -0,0 +1,17 @@
+import { Route, Routes } from "react-router-dom";
+import Dasboard from "../components/admin/admin";
+
+export function PrivateRouter() {
+  return (
+    <Routes>
+      <Route
+        path="dashboard"
+        element={
+          <div className="flex h-screen w-scree bg-[#f2f2f4]">
```

Comment:
Please avoid styling here, please transfer the style to the appropriate component.